server/src/main/java/io/seata/server/coordinator/DefaultCoordinator.java

```diff
@@ -337,7 +338,8 @@ protected void timeoutCheck() {
                     return false;
                 }
 
-                LOGGER.warn("Global transaction[{}] is timeout and will be rollback,transaction begin time:{} and now:{}", globalSession.getXid(), globalSession.getBeginTime(), System.currentTimeMillis());
+                LOGGER.warn("Global transaction[{}] is timeout and will be rollback,transaction begin time:{} and now:{}", globalSession.getXid(),
```

Comment:
The annotation timeout is in milliseconds, here the log format is printed in seconds.